tests/Fixtures/Integration/misc/fully_qualified_strict_types-leading_backslash_in_global_namespace.test

```diff
@@ -0,0 +1,41 @@
+--TEST--
+Integration of fixers: fully_qualified_strict_types /w leading_backslash_in_global_namespace
+--RULESET--
+{
+    "@PhpCsFixer": true,
+    "fully_qualified_strict_types": ["leading_backslash_in_global_namespace": true]
+}
+--EXPECT--
+<?php
+
+/**
+ * @param \DateTime $dateTime
+ * @param int $v
```

Comment:
It is super important to not prepend `\` otherwise `leading_backslash_in_global_namespace: true` is useless/broken.